src/gennylib/src/context.cpp

```diff
@@ -37,10 +38,15 @@ WorkloadContext::WorkloadContext(const YAML::Node& node,
       _rateLimiters{10},
       _poolManager{mongoUri, apmCallback} {
 
+    std::set<std::string> validSchemaVersions{"2018-07-01", "2019-04-01"};
```

Comment:
so actually `2019-04-01` isn't a valid schema version for anything except "inner phases" and in fact it's the same as `2018-07-01` but just a subset of it - I wonder if a name like `2018-07-01.Phase` or something might make more sense?